README.md

```diff
@@ -164,7 +164,10 @@ optional arguments:
   -H, --hidden          include hidden (dot) files [default: off]
   -i, --includeimgp     re-process _IMGP files. * RISKY: refer to docs
   -k, --keep            skip (honors -c or --pr) images matching specified
```

Comment:
Please indicate optional parameter correctly. Currently it appears the option doesn't take any parameter.